docs/06-contributors/999-rfcs/000-stories/story-15.md

```diff
@@ -62,30 +62,20 @@ interface ITaskList {
 
```

Comment:
Unrelated to your changes, but can you please also remove the `void` return type from `ITaskList.remove()`?